actions/views/lhs.js

```diff
@@ -1,15 +1,29 @@
+// Copyright (c) 2018-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
 
 import {fetchMyChannelsAndMembers} from 'mattermost-redux/actions/channels';
 
 import {loadProfilesForSidebar} from 'actions/user_actions.jsx';
 import {loadStatusesForChannelAndSidebar} from 'actions/status_actions.jsx';
 import store from 'stores/redux_store.jsx';
+import {ActionTypes} from 'utils/constants.jsx';
 
-const dispatch = store.dispatch;
 const getState = store.getState;
 
+export const toggle = () => (dispatch) => dispatch({
+    type: ActionTypes.TOGGLE_LHS,
+});
+
+export const open = () => (dispatch) => dispatch({
+    type: ActionTypes.OPEN_LHS,
+});
+
+export const close = () => (dispatch) => dispatch({
+    type: ActionTypes.CLOSE_LHS,
+});
+
 export async function initTeamChangeActions(teamId) {
-    await fetchMyChannelsAndMembers(teamId)(dispatch, getState);
+    await fetchMyChannelsAndMembers(teamId)(store.dispatch, getState);
```

Comment:
`store.getState` ? also you could do `await store.dispatch(fetchMyChannelsAndMembers(teamId));`

Reply:
Gah, you keep suggesting this and I keep forgetting to integrate it into my PRs. Thank you -- I'll clean this up :)

Reply:
As it turns out, this method is entirely unused anyway. I'll push up its removal with any changes required after feedback from @jespino.